app/views/admin/accounts/show.html.haml

```diff
@@ -16,16 +16,16 @@
         .account__header__fields
           - fields.each do |field|
             %dl
-              %dt.emojify{ title: field.name }= Formatter.instance.format_field(account, field.name, custom_emojify: true)
+              %dt.emojify{ title: field.name }= prerender_custom_emojis(h(field.name), account.emojis)
```

Comment:
Same than for `app/views/accounts/_bio.html.haml` regarding links. This might warrant a new helper.